### PR TITLE
added namedtype

### PR DESCRIPTION
**Pull request by qchateau**
Specify library name and version:  **namedtype/20190324**

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/wiki#how-to-submit-a-pull-request) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/namedtype/all/test_package/conanfile.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    generators = "cmake"
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def test(self):
+        if not tools.cross_building(self.settings):
+            self.run(os.path.join("bin", "main"))
```
Comment (madebr) [inline recipes/namedtype/all/test_package/conanfile.py]:
```suggestion
            self.run(os.path.join("bin", "main"), run_environment=True)
```

**Comment by uilianries** [comment]
There a release version there: 1.0.0. Maybe you could request to the author release a new version too.

**Comment by qchateau** [comment]
> There a release version there: 1.0.0. Maybe you could request to the author release a new version too.

I've made a request to the author, do you recommend changing the recipe to package the officially released version ? If the author releases a new version, I can always add it to the recipe later.

**Comment by uilianries** [comment]
No, keep the date. The release version 1.0.0 is outdated. This project is not the first which has only one release and it's outdated. Thanks for reporting the author.

**Pull request merged**